### DISCUSSION TITLE
发布区块一直在等待状态

### DISCUSSION THREAD

**Discussion by FairyShuai** [opentiny/tiny-engine#386] [Q&A]
请问如何发布一个区块呢，点击发布为私有区块，一直显示进度条，并且发布不成功，是什么原因呢
![微信图片_20240408134738](https://github.com/opentiny/tiny-engine/assets/71536307/ee2541e3-764b-406c-8109-79585ddc3f25)

**Comment by wenmine** [comment]
具体要看你启动的webservice里的报错信息，以及是不是正确配置了token，发布到你的私仓里
